crates/ruff/src/lib.rs

```diff
@@ -325,9 +321,14 @@ pub fn check(args: CheckCommand, log_level: LogLevel) -> Result<ExitStatus> {
         printer_flags,
     );
 
+    let preview = overrides.preview.unwrap_or_default().is_enabled();
```

Comment:
Does this respect `preview = true` in configuration files? I _think_ this might only respect `--preview` when it's set on the command line -- is that deliberate?

Reply:
No that was not deliberate, good catch.

Reply:
No worries! Was trying to pin down the build failures on #9599 after merging in `main`, and spotted this :)